src/plugins/GUITestBase/src/GTUtilsOptionPanelMca.cpp

```diff
@@ -190,6 +182,26 @@ void GTUtilsOptionPanelMca::pushExportButton(HI::GUITestOpStatus &os) {
 }
 #undef GT_METHOD_NAME
 
+#define GT_METHOD_NAME "showAlternativeMutations"
+void GTUtilsOptionPanelMca::showAlternativeMutations(HI::GUITestOpStatus &os, bool show, int value, bool withSpinbox, QWidget *parent) {
+    GTUtilsOptionPanelMca::openTab(os, Tabs::Reads, parent);
+    GTGroupBox::setChecked(os, "mutationsGroupBox", show, parent);
+    if (!show) {
+        GTGlobals::sleep(1000);
```

Comment:
Do not use sleeps in tests (unless the case is out of QT logic scope)

Reply:
OK